Toolset/libraries/revshortcutslibrary.livecodescript

```diff
@@ -66,7 +66,7 @@ on arrowKey pDirection, pTarget
    put revTargetStack(line 1 of (the selObj)) into tStack
    
    if (tStack is line 1 of the openStacks or tStack is the short name of the topStack) \
-         and word 1 of (the selobj) is not "card" then
+         and word 1 of (the selobj) is not "card" and word 1 of (the selobj) is not "stack" then
```

Comment:
actually it can... create a stack, type `select this stack; put the selObj` in the msg box